src/main/flight/gyroanalyse.c

```diff
@@ -138,14 +180,12 @@ void gyroDataAnalyseStateInit(gyroAnalyseState_t *state, uint32_t targetLooptime
     arm_rfft_fast_init_f32(&state->fftInstance, FFT_WINDOW_SIZE);
 
 //    recalculation of filters takes 4 calls per axis => each filter gets updated every DYN_NOTCH_CALC_TICKS calls
+//    at 4khz gyro loop rate this means 8khz / 4 / 3 = 666Hz => update every 1.5ms
```

Comment:
"at 8kHz gyro loop rate"

Actually all khz should be replaced by kHz.

Reply:
Thanks I'll do that - my bad